fastddsspy_tool/src/cpp/tool/Controller.cpp

```diff
@@ -439,24 +496,27 @@ void Controller::help_command_(
-            << "\tparticipants verbose   : verbose information about DomainParticipants discovered in the network.\n"
-            << "\tparticipants <Guid>    : verbose information related with a specific DomainParticipant.\n"
-            << "\twriters                : DataWriters discovered in the network.\n"
-            << "\twriters verbose        : verbose information about DataWriters discovered in the network.\n"
-            << "\twriters <Guid>         : verbose information related with a specific DataWriter.\n"
-            << "\treader                 : DataReaders discovered in the network.\n"
-            << "\treader verbose         : verbose information about DataReaders discovered in the network.\n"
-            << "\treader <Guid>          : verbose information related with a specific DataReader.\n"
-            << "\ttopics                 : Topics discovered in the network.\n"
-            << "\ttopics verbose         : verbose information about Topics discovered in the network.\n"
-            << "\ttopics <name>          : verbose information related with a specific Topic.\n"
-            << "\tshow <name>            : data of a specific Topic (Data Type must be discovered).\n"
-            << "\tshow <name> verbose    : data with additional source info of a specific Topic.\n"
-            << "\tshow all               : verbose data of all topics (only those whose Data Type is discovered).\n"
+            << "\thelp                            : this help.\n"
+            << "\tversion                         : tool version.\n"
+            << "\tquit                            : exit interactive CLI and close program.\n"
+            << "\tparticipants                    : DomainParticipants discovered in the network.\n"
+            << "\tparticipants verbose            : verbose information about DomainParticipants discovered in the network.\n"
+            << "\tparticipants <Guid>             : verbose information related with a specific DomainParticipant.\n"
+            << "\twriters                         : DataWriters discovered in the network.\n"
+            << "\twriters verbose                 : verbose information about DataWriters discovered in the network.\n"
+            << "\twriters <Guid>                  : verbose information related with a specific DataWriter.\n"
+            << "\treader                          : DataReaders discovered in the network.\n"
+            << "\treader verbose                  : verbose information about DataReaders discovered in the network.\n"
+            << "\treader <Guid>                   : verbose information related with a specific DataReader.\n"
+            << "\ttopics                          : Topics discovered in the network in compact format.\n"
+            << "\ttopics v                        : Topics discovered in the network.\n"
+            << "\ttopics vv                       : verbose information about Topics discovered in the network.\n"
```

Comment:
Idea for the future: it would be helpful to have a "types" command that would be able to print the IDL representation when the TypeObject has been received.

Reply:
Yes! I thought the same when I was reviewing the _topics_ command. See [21850](https://eprosima.easyredmine.com/issues/21850)